.dockerignore

```diff
@@ -5,6 +5,7 @@ run_tests.sh
 
 # Byte-compiled / optimized / DLL files
 __pycache__/
```

Comment:
I think this is now part of the next line as well, as ** can also match zero directories.